lang/nl/route.php

```diff
@@ -13,5 +13,6 @@
     "checkout" => "afrekenen",
     "complete_order" => "voltooien",
     "cart" => "winkelwagen",
+    "manage-orders" => 'beheer-bestellingen',
```

Comment:
```suggestion
    "orders" => 'bestellingen',
```
Aangezien de route nu zo is: http://127.0.0.1:8000/beheren/beheer-bestellingen